### ISSUE TITLE
Create a spreadsheet in the builds with rule information

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#2004]
Need columns for ruleid, group message, rule message, violation level, toolkit level, WCAG requirements

**Comment by philljenkins** [comment]
The spreadsheet should have multiple lines in some cases to **_make it easier for sorting and comparing_**:
- Some rules have _multiple_ fail messages, so those rules should have _multiple_ lines in the spreadsheet.
- Some rules map to _multiple_ WCAG SC's, so those rules should have _multiple_ lines
- Some rules map to ACT , so that column should be added.

**Comment by philljenkins** [comment]
@tombrunet See also the following spreadsheetsb for use cases & needs
[ACT Master 2024.xlsx](https://ibm.box.com/s/evec4kfixgzjw8tmghnr9f13xxw99e33)
[Master-Checker-Rules-IBM-levels.xlsx](https://ibm.box.com/s/nzmxmxhgh2pxkvp2na3xt2xg1qccajmn)

**Comment by MHoov** [comment]
I have a call scheduled on Friday to discuss what we have in mind.

**Comment by MHoov** [comment]
@philljenkins lets discuss what this concept can cover before you start adding too many things please. This is intended to specifically address your comment last week about how difficult it was to review rules for consistency - and the files you provided to me in slack. This is not the larger concept in issue 6542 (Checker automated sync for rule to help requirements mapping)

**Comment by MHoov** [comment]
This what we discussed in the mural [Consistency Map](https://app.mural.co/t/ibmaccessibility7766/m/ibmaccessibility7766/1723838753553/1639f330ea5190be7378c93cc960797b5cb48f60?sender=ucf3511eeea158b2c60257653)

![Image](https://github.com/user-attachments/assets/e70db028-f3a0-4f76-a902-71db0ccd5e8c)

**Comment by philljenkins** [comment]
Blocking #2030

**Comment by MHoov** [comment]
See related PR #2071

**Comment by philljenkins** [comment]
@tombrunet also added "Pass" messages to spreadsheet.
and it's automatically generated during PR build process as an Artifact in Actions
[Rule spreadsheet](https://github.com/IBMa/equal-access/actions/runs/11338847065/artifacts/2056054302)

I also recommend changing the file name 
from: **_Rule spreadsheet_**
to: **_Rules mapping_**
